bazel/repository_locations.bzl

```diff
@@ -490,6 +490,18 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         release_date = "2020-07-10",
         cpe = "cpe:2.3:a:nodejs:node.js:*",
     ),
+    com_github_nodejs_llhttp = dict(
+        project_name = "llhttp",
+        project_desc = "Parser for HTTP messages written in C",
+        project_url = "https://github.com/nodejs/llhttp",
+        version = "3.0.1",
+        sha256 = "6ae898e5c5e8081be68d4d93d24349c8a4f4d0c7e3eb89f0e35db5a7f15d59fa",
+        strip_prefix = "llhttp-release-v{version}",
+        urls = ["https://github.com/nodejs/llhttp/archive/release/v{version}.tar.gz"],
+        use_category = ["dataplane_core"],
+        release_date = "2020-02-16",
```

Comment:
Wondering if we should use the more current `4.0.0` release - https://github.com/nodejs/llhttp/releases/tag/release%2Fv4.0.0 ? `3.0.1` is marked as the latest release however. I'll try and ping @indutny to get guidance on what we should target

Reply:
Sorry for confusion. 4.0.0 is the latest version, I just had to do an extra backport release in 3.x branch.

Reply:
thank you both for that! will update.